net-misc/onedrivegui/onedrivegui-1.0.3-r2.ebuild

```
@@ -4,43 +4,66 @@
 EAPI=8
 
 M_PN=OneDriveGUI
-
-DISTUTILS_USE_PEP517=setuptools
+DISTUTILS_USE_PEP517=no
 PYTHON_COMPAT=(python3_{9..13})
 
-inherit desktop distutils-r1
+inherit desktop distutils-r1 xdg-utils
 if [[ ${PV} == "9999" ]]; then
 	EGIT_REPO_URI="https://github.com/bpozdena/${M_PN}.git"
 	inherit git-r3
 else
-	SRC_URI="https://github.com/bpozdena/${M_PN}/archive/refs/tags/v${PV}.tar.gz -> ${PN}-${PV}.tar.gz"
-	KEYWORDS="~amd64"
+	SRC_URI="https://github.com/bpozdena/${M_PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
     S="${WORKDIR}/${M_PN}-${PV}"
 fi
 
 DESCRIPTION="A simple GUI for OneDrive Linux client, with multi-account support."
 HOMEPAGE="https://github.com/bpozdena/OneDriveGUI"
+LICENSE="GPL-3"
+SLOT="0"
+
+RESTRICT=test
 
 RDEPEND="<net-misc/onedrive-2.5
-	dev-python/requests
-	dev-python/pyside6[gui(+),webengine(+),widgets(+)]
+    !!net-misc/onedrivegui-bin
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/pyside6[gui(+),webengine(+),widgets(+),${PYTHON_USEDEP}]
```

`pyside6` has been renamed bec
```suggestion
	dev-python/pyside[gui(+),webengine(+),widgets(+),${PYTHON_USEDEP}]
```